### ISSUE TITLE
Redux implementation

### ISSUE THREAD

**Issue by addyosmani** [tastejs/todomvc#1674]
Source: https://github.com/reactjs/redux/tree/master/examples/todomvc
Contributors: @gaearon @jugend 

Hey folks, would either of you be interested in PR'ing in a Redux implementation to TodoMVC? I know you maintain your own version over on the redux repo but just wanted to check ⛵ The main benefit is listing on the site so folks can compare with other implems.

What some folks do is continue updating their primary source in their own repo with periodic syncs back to here while others prefer to just link to the upstream version. We would be fine with either (and would also be totally 😎 if there wasn't interest in this atm).

**Comment by fibo** [comment]
As many others, I also created a Redux example here: https://github.com/fibo/redux-todoapp

It uses a router, async actions and has an API server (implemented by [json-server](https://github.com/typicode/json-server) which is really nice :)

However, I think Redux examples should not merged in this repository, if they need a server to run. They cannot be added to the http://todomvc.com/ website as demos.
Maybe the best options is to add links to them in a wiki page or even http://todomvc.com/examples/react/

@petehunt your final thought, and thanks for sharing this collection of demos, I used it as a starting point for a course, I owe you a `Beer` Component.
